### PR TITLE
Feature/lji 33 add notification microservice

### PR DESCRIPTION
**Pull request by Cioperis**

### CODE REVIEW

File: src/main/java/com/coherentsolutions/pot/insurance/dto/ClaimDTO.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public class ClaimDTO implements Serializable {
 
   private String claimNumber;
 
-  private String employee;
+  private String employeeUserName;
```
Comment (anton-kolyaev) [inline src/main/java/com/coherentsolutions/pot/insurance/dto/ClaimDTO.java:27]: that's not ok to update already released API without versioning, is it really necessary? if it is then create a new version for endpoint and a new version of DTO (leave old one for compability)

File: src/main/java/com/coherentsolutions/pot/insurance/service/EmployeeService.java [hunk 2]
```diff
@@ -62,33 +61,43 @@ public List<EmployeeDTO> getAllEmployees() {
...
+
+      // Send notification
+      String message = """
+          Dear %s,
+
+          Your account has been deactivated.
+
+          Best regards,
+          Your Company
+          """.formatted(employee.getFirstName());
```
Comment (anton-kolyaev) [inline src/main/java/com/coherentsolutions/pot/insurance/service/EmployeeService.java:91]: Looks like constant. 
And we potentially will want to have internatilization/localization (i18n/l10n) - let's discuss on daily meeting

File: notif-svc/src/test/java/com/coherentsolutions/pot/insurance/NotificationsMicroserviceApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.coherentsolutions.pot.insurance;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class NotificationsMicroserviceApplicationTests {
+
+	@Test
+	void contextLoads() {
```
Comment (anton-kolyaev) [inline notif-svc/src/test/java/com/coherentsolutions/pot/insurance/NotificationsMicroserviceApplicationTests.java:10]: Please add some meaningful integration test

**Review by anton-kolyaev — CHANGES_REQUESTED** [review]

File: src/main/java/com/coherentsolutions/pot/insurance/dto/ClaimDTO.java (hunk 1 above)
Comment (Cioperis) [reply src/main/java/com/coherentsolutions/pot/insurance/dto/ClaimDTO.java:27]: Saulius asked me to update the DTO, do I still need to create a separate one?

File: src/main/java/com/coherentsolutions/pot/insurance/service/EmployeeService.java (hunk 2 above)
Comment (Cioperis) [reply src/main/java/com/coherentsolutions/pot/insurance/service/EmployeeService.java:91]: added notic properties

File: notif-svc/src/test/java/com/coherentsolutions/pot/insurance/NotificationsMicroserviceApplicationTests.java (hunk 3 above)
Comment (Cioperis) [reply notif-svc/src/test/java/com/coherentsolutions/pot/insurance/NotificationsMicroserviceApplicationTests.java]: done

**Review by anton-kolyaev — APPROVED** [review]

**Review by NormantasN — APPROVED** [review]

**Review by SavaLX — APPROVED** [review]

**Pull request merged**